operations/patch_util.go

```diff
@@ -31,7 +32,9 @@ var patchDisplayTemplate = template.Must(template.New("patch").Parse(`
     Description : {{if .Patch.Description}}{{.Patch.Description}}{{else}}<none>{{end}}
 	  Build : {{.Link}}
 	 Status : {{.Patch.Status}}
+{{if .ShowFinalized}}
```

Comment:
for the `ShowFinalized == true` case will this add newlines around it?

Reply:
It adds too many new lines. Fixing.

Reply:
Fixed.